.github/workflows/unstale.yml

```diff
@@ -0,0 +1,19 @@
+name: Unmark issues and pull requests as stale on activity
+on:
+  issue_comment:
+    types: [created]
+
+# actions/stale does this automatically, but only once a day.
+# This immediately removes the label when the user creates a comment.
+jobs:
+  remove-stale-label:
+    if: (github.repository == 'rubocop/rubocop')
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Remove stale label
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
Is it necessary to set this `env` variable?

> To use gh in GitHub Actions, add `GH_TOKEN: ${{ github.token }}` to `env`.

https://cli.github.com/manual/gh_auth_login

Reply:
Yeah, this is necessary: https://github.com/Earlopain/rubocop/actions/runs/13919611046/job/38949572852